### DISCUSSION TITLE
Slideshow enhancement

### DISCUSSION THREAD

**Discussion by TNO92** [LycheeOrg/Lychee#2425] [Ideas] [answered]
### Enhancement

Slideshow is a great feature ! thanks a lot

Missing some little awesome enhancements : 

- loop or not
- choose order → by name, by import date, by picture date and random
- display exif or not
- time in second for each image
- put these parameters in the url to share it ex. : https://MYLYCHEE.fr/frame/obZ3mw4z_9IxuyJWXcwHY9Gb?order=byName&exif=true&timeInS=10&loop=false

Thomas

**Comment by ildyria** [comment, marked as answer]
I made it differently.
On v6, you can just press "space" on an album or photo view and it will trigger the slide show.

- the order is defined by the order of pictures in the album.
- Loop is defined by the global configuration.
- exif can be rotated with `o`.
- the refresh time is defined in global configuration.
